wallet/public/index.html

```diff
@@ -3,6 +3,28 @@
   <head>
     <meta charset="utf-8" />
     <script src="%PUBLIC_URL%/lockdown.umd.js"></script>
+    <script>
+      const devUnsafe = '%NODE_ENV%' === 'development' ? 'unsafe' : 'safe';
+
+      const { pow: mathPow } = Math;
+      Math.pow = (base, exp) =>
+        typeof base === 'bigint' && typeof exp === 'bigint'
+          ? base ** exp
+          : mathPow(base, exp);
+
```

Comment:
These are sort of fresh lines. Might want to omit https://github.com/Agoric/wallet-app/issues/84 
```suggestion
```

Reply:
Sure, done. To be clear, in this PR I just restored this file from before the recent keplr change, and then changed taming to default to safe per @erights previous comments.